src/AlixBa/Addic7edSubtitles/Helpers/Episode.php

```diff
@@ -51,8 +51,8 @@ public function __construct($episodeFilename)
     {
         preg_match($this->pattern, $episodeFilename, $matches);
 
-        $this->showName          = $matches['showname'];
-        $this->sanitizedShowName = self::sanitizeShowName($matches['showname']);
+        $this->showName          = self::capitalizeShowName($matches['showname']);

```

Comment:
I think we can do all the process in sanitize and keep showName to display it when the script is running. It will allow the end user to see if the extracted showName really matches the show name.